packages/flutter/test/cupertino/form_section_test.dart

```diff
@@ -168,4 +168,57 @@ void main() {
     final Iterable<RenderClipRRect> renderClips = tester.allRenderObjects.whereType<RenderClipRRect>();
     expect(renderClips, isEmpty);
   });
+  
```

Comment:
Looks like the linter is complaining about a white space here.

Reply:
Cheers, fixed both and all good now 👍